### ISSUE TITLE
Icon request: fa-tidal

### ISSUE THREAD

**Issue by robinwellstrom** [FortAwesome/Font-Awesome#8850]
![tidal-icon-logo](https://cloud.githubusercontent.com/assets/9363426/14111140/d9a758ca-f5c9-11e5-8a5f-25e88147aff2.png)

The "new" music streaming tool. Working with a lot of artist websites that asks me to attach this icon next to spotify, itunes, social networks etc.

**Comment by breakthecode** [comment]
Oh yes! I need this icon ASAP for a project I´m working on right now...

Please please have this attached ASAP. :)

**Comment by RASK901** [comment]
![tidal](https://cloud.githubusercontent.com/assets/18583587/14710296/1b1b303e-07d5-11e6-9651-b90c561f3ce6.png)
[tidal.zip](https://github.com/FortAwesome/Font-Awesome/files/229965/tidal.zip) (SVG File)

+1

**Comment by samkite** [comment]
+1

**Comment by CaliforniaRex** [comment]
+1

**Comment by paul-HPA** [comment]
+1 , thank you!

**Comment by kylefthomas** [comment]
+1 - please add this

**Comment by part4git** [comment]
+1 - yes please.

**Comment by haleyngonadi** [comment]
+1 :)

**Comment by TLMcode** [comment]
thanks for the vector...

**Comment by Ekeler** [comment]
+1

**Comment by maxharris** [comment]
+1

**Comment by packrobottom** [comment]
how is this still not added in nov 2017?

**Comment by DreamEmulator** [comment]
Please add

**Comment by martindefatte** [comment]
+1

**Comment by prknje** [comment]
+1

**Comment by grlindburg** [comment]
Is this ever going to happen? Really super annoying when all I wanna do is add it to a page like this: liwb.benburden.com

**Comment by FredDaniel** [comment]
I also would love to have a Tidal icon available (https://tidal.com).
Why is that this request constantly is being ignored or dropping off ?

**Comment by maxharrisnet** [comment]
Ya'll are trash for this. I am not using FA anymore.

**Comment by davegandy** [comment]
@maxharrisnet Seems like you might be having a bad day. No need to take it out on folks giving something away for free. I'd ask you to re-consider your approach to getting your way. Definitely won't work here.

In addition to the way you're treating folks, I'd ask you to reconsider your entitlement attitude as well. Gratitude tends to go quite a bit farther.

Additionally, this serves as your final warning for breaking our [Code of Conduct](https://github.com/FortAwesome/Font-Awesome/blob/master/CODE_OF_CONDUCT.md).

Finally, I hope everything is okay in your life right now. I honestly wish you all the best.

**Comment by davegandy** [comment]
For folks asking why this has not been implemented yet, we typically add community requested icons by how many folks are asking for it. This one just hasn't popped the stack yet. More details are in our [Icon Leaderboard](https://fontawesome.com/community/leaderboard/brands).

**Comment by maxharrisnet** [comment]
@davegandy You are right, sorry to hurt your feelings. The frustration is more about the lack of response/information on why these requests are not being honored. I didn't know about the leaderboard until now, but even most of the top 10 requests have been up for years and there are frustrated comments in every thread. Why is my terrible comment the only one that gets a response?

Here's my constructive criticism:
- The leaderboard is not mentioned anywhere in the process (aside from the link in the footer). I didn't know about it until you just told me, and I am probably not alone in that. I suggest including some information for users either on the main site, in Github, or in the automated email.
- Hire more people to make fonts so that it doesn't take years to make a new one
- Hire more people to respond to comments
- Respond to people when they are being positive. Don't wait for jerks like me to get your attention. Make your users feel heard.
- Follow your own code of conduct

You have a great platform that people like. Don't underestimate the value of customer service.

**Comment by heismauri** [comment]
+1

**Comment by belinecidral** [comment]
+1

**Comment by orangemooncreative** [comment]
+1

**Comment by kylerchin** [comment]
+1

**Comment by kylerchin** [comment]
+1

**Comment by kylerchin** [comment]
I NEED THIS ASAP, I need to put this on my website, please approve!

**Comment by Maddieman** [comment]
+1

**Comment by kylerchin** [comment]
Is this still alive? I still need thisl,

**Comment by warmwhisky** [comment]
+1

**Comment by orangemooncreative** [comment]
i'm really perplexed that we are still asking for this icon. anyone who is building anything music / streaming related needs this in their icon set. you all just released an entire set of music related icons and didn't include this in the brand icon update. how? why?

**Comment by waspeer** [comment]
+1

**Comment by sanek4life** [comment]
+1

**Comment by dwoznicka** [comment]
+1 !

**Comment by makemagik** [comment]
+1

**Comment by Yizack** [comment]
+1

**Comment by WrestlingMedia** [comment]
+1

**Comment by ZeroEpoch1969** [comment]
+1 Definitely needed.

**Comment by tagliala** [comment]
@ZeroEpoch1969 please take a look at our [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work

**Comment by DrewMcArthur** [comment]
bump, is there a reason this has yet to be added?

**Comment by tagliala** [comment]
@DrewMcArthur as per my latest comment, please take a look at our [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work

**Comment by dweuste** [comment]
Another bump for Tidal especially for 6.0

**Comment by maxjutras** [comment]
+1 for tidal

**Comment by jminguely** [comment]
+1

**Comment by fdioni** [comment]
+1 for tidal

**Comment by ribkix** [comment]
+1 PLEASE ADD THIS

**Comment by brewermfnyc** [comment]
+1 !! Yes please!!

**Comment by netmcgill** [comment]
+1 on TIDAL please! pretty please

**Comment by MaraLanz** [comment]
+1 on TIDAL too. How long until we see this icon in the "in progress" board?

**Comment by makemagik** [comment]
6 years later I still need the Tidal icon. Lets go! +1

**Comment by JonathonDunford** [comment]
Seems like this is a waste of time, but +1.

Hire a designer or cheap VA and the list of top 50 would be done in a few days.

**Comment by dixls** [comment]
+1

**Comment by makemagik** [comment]
Bonjour,

Veuillez prendre note que je suis en vacances du 1er au 21 août inclusivement. 

Pour toute urgence, veuillez envoyer un courriel a ***@***.*** 

Merci et passez un bel été!

**Comment by tristan-smith** [comment]
+1

**Comment by MrHedgehogEsquire** [comment]
8 Years and no Tidal yet?
+1

**Comment by adygreg** [comment]
+1 

for my website eddierating.com :,(

**Comment by Nickessss** [comment]
+1

when...?

**Comment by jackammon** [comment]
+1 ... 8 years later ...

**Comment by wont-stream** [comment]
Another +1, need to see this FA.

**Comment by EvyBongers** [comment]
+1

**Comment by gnireorb** [comment]
+1

**Comment by killparty** [comment]
+1

**Comment by killparty** [comment]
+1

**Comment by deadflowers** [comment]
+1 

Wild that this has been open since 2016 as pivotal a role as they've played in streaming and championing the consumer

**Comment by osvalda** [comment]
+1

**Comment by cellcoresystems** [comment]
+1 Please add this it is really needed - thx

**Comment by thenomi** [comment]
oh wow it's been nearly 10 years? yes please.

**Comment by makemagik** [comment]
Bonjour,
Je serai en vacances du 7 au 25 juillet inclusivement.
Pour toute urgence, vous pouvez m’envoyer un texto au (819) 993-8115.
Merci de votre compréhension et au plaisir de vous retrouver à mon retour.

**Comment by killparty** [comment]
> oh wow it's been nearly 10 years? yes please. 

Yeah it's embarrassing

**Comment by tagliala** [comment]
Hello, this has been released in Fa 7.0

https://fontawesome.com/icons/tidal?f=brands&s=solid
